core/frontend/src/main/scala/io/udash/utils/FileUploader.scala

```diff
@@ -18,8 +18,12 @@ class FileUploader(url: Url) {
     )
 
   /** Uploads provided `files` in a field named `fieldName`. */
-  def upload(fieldName: String, files: Seq[File],
-             extraData: Map[js.Any, js.Any] = Map.empty): ReadableModelProperty[FileUploadModel] = {
+  def upload(
+    fieldName: String,
+    files: Seq[File],
+    extraData: Map[js.Any, js.Any] = Map.empty,
+    successfulResponseCallback: Option[HttpResponse => Unit] = None
```

Comment:
I think that we should add `Option[HttpResponse]` to `FileUploadModel` and set this value similary to `state` field instead of providing this callback.